src/components/nodes/intermediateMessageCatchEvent/index.js

```diff
@@ -1,6 +1,9 @@
 import component from './intermediateMessageCatchEvent.vue';
 import omit from 'lodash/omit';
 import { configId } from '@/components/inspectors/configId';
+import NodeIdGenerator from '../../../NodeIdGenerator';
+
+const generateMessageEventId = new NodeIdGenerator();
```

Comment:
`generateMessageEventId` sounds like the name of a method. To keep this consistent with how it's named in `Modeler.vue`, I think a name of `messageEventIdGenerator` would be better. Also, since it seems the same instance will always use the same prefix, I think it would be better to pass the prefix into the constructor as well (maybe have the class handle adding the `_` separator as well):

```js
const messageEventIdGenerator = new NodeIdGenerator('message_event');
...
id: messageEventIdGenerator.generateUniqueNodeId(),
```

You should also call `generateUniqueNodeId` and **not** `generateNewNodeId`, to ensure IDs are unique. Because right now, if I drag an Intermediate Message Catch Event and manually change it's Message Event Identifier to `message_event_2`, the next Intermediate Message Event I add will be given the same ID.